tools/fileinfo/bugs/upx-detection-segfault/test.py

```diff
@@ -0,0 +1,11 @@
+from regression_tests import *
+
+class TestUpxDetectionSegfault(Test):
```

Comment:
Consider mentioning the ticket (https://github.com/avast/retdec/issues/821) in a comment so we know the origin of the test.